CTestCustom.cmake

```diff
@@ -20,6 +20,13 @@ if(CTEST_BUILD_NAME MATCHES slc6|centos7)
               roottest-root-multicore-fork)
 endif()
 
+if(WIN32)
```

Comment:
I would prefer:
```suggestion
if(MSVC)
```

Reply:
> `if(MSVC)`

I tried that first, but for some reason, it didn't work.  :-/  Leaving as `if(WIN32)` instead.